Tabs.php

```diff
@@ -240,8 +244,11 @@ protected function renderDropdown($itemNumber, &$items, &$panes)
             if (isset($item['visible']) && !$item['visible']) {
                 continue;
             }
-            if (!array_key_exists('content', $item)) {
-                throw new InvalidConfigException("The 'content' option is required.");
+            if (!(array_key_exists('content', $item) xor array_key_exists('url', $item))) {
```

Comment:
okay, for better understanding, the error message should be `Either the 'content' or the 'url' option is required, but only one can be set.` then.